data/sql/updates/pending_db_world/add-terrorclaw-fear-spell.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 20477;
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 20477);
```

Comment:
It already has SmartAI in the template so it doesn't need this line.
```suggestion
--
DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 20477);
```